VARIABLES.md

```diff
@@ -502,15 +518,15 @@ Default:  "{{confluent_server_enabled}}"
 
 ### schema_registry_user
 
-Only use to customize Linux User Schema Registry Service runs with. User must exist on host.
+Only use to customize Linux User Schema Registry Service runs with. Default user is cp-schema-registry
 
 Default:  "{{schema_registry_default_user}}"
 
 ***
 
 ### schema_registry_group
 
-Only use to customize Linux Group Schema Registry Service user belongs to. Group must exist on host.
+Only use to customize Linux Group Schema Registry Service user belongs to. Default group is confluent
```

Comment:
Modify this variable to customize the Linux Group that the Schema Registry Service user belongs to. Default group is confluent.